apps/hubspot/functions/createModules.ts

```diff
@@ -76,3 +90,84 @@ const createModuleFile = async (
     );
   }
 };
+
+const getFields = (field: SdkField): string => {
+  const { type } = field;
+  let fields;
+  switch (type) {
+    case 'Symbol':
+      fields = structuredClone(TEXT_FIELD_TEMPLATE);
+      if (field.value) fields[0].default = field.value;
+      break;
+    case 'RichText':
+      fields = structuredClone(RICH_TEXT_FIELD_TEMPLATE);
+      if (field.value) fields[0].default = documentToHtmlString(field.value);
+      break;
+    case 'Number':
+    case 'Integer':
+      fields = structuredClone(NUMBER_FIELD_TEMPLATE);
+      if (field.value) fields[0].default = field.value;
+      break;
+    case 'Date':
+      const value = field.value as string;
+      if (!value) {
+        fields = structuredClone(DATETIME_FIELD_TEMPLATE);
+      } else if (value.includes('T')) {
+        fields = structuredClone(DATETIME_FIELD_TEMPLATE);
+        fields[0].default = new Date(value).getTime();
+      } else {
+        fields = structuredClone(DATE_FIELD_TEMPLATE);
+        fields[0].default = new Date(value).getTime();
+      }
+      break;
+    case 'Location':
+      fields = structuredClone(TEXT_FIELD_TEMPLATE);
+      if (field.value) fields[0].default = `lat:${field.value.lat}, long:${field.value.lon}`;
+      break;
+    case 'Array':
+      fields = structuredClone(TEXT_FIELD_TEMPLATE);
+      if (field.value) fields[0].default = field.value.join(', ');
+      break;
+    case 'Link':
+      fields = structuredClone(IMAGE_FIELD_TEMPLATE);
+      if (field.value) {
+        fields[0].default.src = field.value.url;
+        fields[0].default.width = field.value.width;
+        fields[0].default.height = field.value.height;
+      }
+      break;
+    default:
+      fields = structuredClone(TEXT_FIELD_TEMPLATE);
+      if (field.value) fields[0].default = field.value;
+      break;
```

Comment:
💭 If you reach here, you should probably fail because we are not supporting that kind of field